releasenotes/notes/57504.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+  - 57504
+releaseNotes:
+  - |
+    **Fixed** spurious failures in Helm-based upgrade tests by skipping schema validation when installing previous-release Gateway charts that reject current values due to stricter JSON schema. Validation remains enabled for current charts under test.
```

Comment:
Given we're fixing tests, I don't think a release note is necessary here.

Reply:
Reverted in https://github.com/istio/istio/pull/57507/commits/81c46ebae5d6feeb78e64e8401e2155d760c4ec1